### PR TITLE
requirements.txt

### PR DESCRIPTION
**Pull request by dmadisetti**
`pip install -r requirements.txt`

### CODE REVIEW

**Comment by davidmann4** [comment]
image
ohh wow you should really change your pw now :)

**Comment by dmadisetti** [comment]
@davidmann4 Haha, if they were real credentials

**Comment by dmadisetti** [comment]
@Mila432 I noticed we both patched the datetime thing at the same time. Is there something I can work on so we don't step on each other's toes?

**Comment by davidmann4** [comment]
really would like to help aswell @Mila432 lets make a chat or something? 

What do you guys think about a pokemon-go-proxy ? this would make working with the API easier:
* Client connects to Proxy
* Proxy shows packets and dumps them into file (for analysis and learning the protocol) 
* Bot connects to Proxy
* Proxy forwards response to App and Bot - which allows us to Watch Bots playing the Game

related: 
* https://github.com/praetorian-inc/mitm-vm
* http://max.kellermann.name/projects/uoproxy/

**Comment by Mila432** [comment]
@davidmann4 this breaks the whole sense of automated scripts and since you are working with the bot it is the direct proxy because you tell it what to do

**Comment by dmadisetti** [comment]
I back @Mila432 , standalone would be nicer

**Comment by davidmann4** [comment]
you are totally right - the bot should be able to run without any external programs.

pokemon-go-proxy is a developer tool designed to debug bots:
* watch them run around and play the game
* trigger specific packets and analyse them to add them to the bot
* etc ...

in my past bot related projects this was the best tool to have

**Comment by Mila432** [comment]
@davidmann4 sorry not interested, i have 100% knowledge about the packets and a full overview

**Comment by davidmann4** [comment]
please merge

**Comment by Mila432** [comment]
you cant use this:
 export POKE_GUSER=
 export POKE_GPASS=
on windows

**Comment by dmadisetti** [comment]
@Mila432 It's `set var=value` on windows. Added to readme

File: main.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def start_work(access_token,ltype):
 			print('[+] Item Storage: {}'.format(profile.item_storage))
 			for curr in profile.currency:
 				print('[+] {}: {}'.format(curr.type, curr.amount))
+			start_private_show(access_token,ltype)
```
Comment (dmadisetti) [inline main.py]: Ahh crap

**Comment by davidmann4** [comment]
still no merge ?

**Comment by Mila432** [comment]
@davidmann4 because this is not windows friend , no windows user will use "set ..."

**Comment by DavisMiculis** [comment]
Instead of the ENV variables, it's a lot easier to use args, for exampe:
`python main.py -u pikachu -p qwert123 -auth google` etc.

**Comment by dmadisetti** [comment]
Merge probs now, and no really point. Better APIsout there now

**Comment by davidmann4** [comment]
https://github.com/rubenvereecken/pokemongo-api
https://pkre.slack.com/messages/reversing/
https://shielded-earth-81203.herokuapp.com/

because they work together :)

**Pull request closed without merging**